java/mdsobjects/src/test/java/MDSplus/MdsTreeTest.java

```diff
@@ -44,12 +44,14 @@ public void testData(){
 		tree = new MDSplus.Tree("java_test0",-1,"READONLY");
 		Assert.assertEquals(true, tree.isReadOnly() );
 		tree.close();
+		tree.close();
+		tree.edit();
```

Comment:
what is this supposed to test.

Reply:
the many close attempts are intentional.  The finalize calls the close too.